examples/policies_configuration_guide.py

```diff
@@ -0,0 +1,198 @@
+    site_list.add_site_range((300, 400))
+    site_list.add_sites({200, 202})
+    site_list_id = api.lists.create(site_list)
+    created_items.append((SiteList, site_list_id))
+
+    # Configure App Probe Class
+    from vmngclient.models.policy.lists import AppProbeClassList, ClassMapList
+
+    class_map = ClassMapList(name="MyClassMap")
+    class_map.assign_queue(1)
+    class_map_id = api.lists.create(class_map)
+    created_items.append((ClassMapList, class_map_id))
+
+    app_probe_class = AppProbeClassList(name="MyAppProbeClass")
+    app_probe_class.assign_forwarding_class("MyClassMap").add_color_mapping("3g", 5)
+    app_probe_class_id = api.lists.create(app_probe_class)
+    created_items.append((AppProbeClassList, app_probe_class_id))
+
+    # Configure SLA Class
+    from vmngclient.models.policy.lists import SLAClassList
+
+    sla_class = SLAClassList(name="MySLAClass")
+    sla_class.assign_app_probe_class(app_probe_class_id, latency=10, loss=1, jitter=5)
+    sla_class.add_fallback_jitter_criteria(10)
+    sla_class.add_fallback_loss_criteria(5)
+    sla_class_id = api.lists.create(sla_class)
+    created_items.append((SLAClassList, sla_class_id))
+
+    # Configure TLOC
+    from vmngclient.models.policy.lists import TLOCList
+
+    tloc_list = TLOCList(name="MyTLOCList")
+    tloc_list.add_tloc(IPv4Address("10.0.0.55"), color="blue", encap="gre")
+    tloc_list.add_tloc(IPv4Address("10.0.0.56"), color="silver", encap="ipsec", preference=5678)
+    tloc_list_id = api.lists.create(tloc_list)
+    created_items.append((TLOCList, tloc_list_id))
+
+    # Configure Region
+    from vmngclient.models.policy.lists import RegionList
+
+    region_list = RegionList(name="MyRegions")
+    region_list.add_regions({1, 2})
+    region_list.add_region_range((3, 6))
+    region_list_id = api.lists.create(region_list)
+    created_items.append((RegionList, region_list_id))
+
+    # Configure VPN
+    from vmngclient.models.policy.lists import VPNList
+
+    vpn_list = VPNList(name="MyVPNList")
+    vpn_list.add_vpns({100, 200})
+    vpn_list.add_vpn_range((1000, 2000))
+    vpn_list_id = api.lists.create(vpn_list)
+    created_items.append((VPNList, vpn_list_id))
+
+    return created_items
+
+
+def run_demo(args: CmdArguments):
+    from vmngclient.session import create_vManageSession
+
+    with create_vManageSession(url=args.url, port=args.port, username=args.user, password=args.password) as session:
+        api = session.api.policy
+        """1. Configure Groups of Interest for Centralized Policy"""
+        groups_of_iterest = configure_groups_of_interest(api)
+        """Cleanup"""
+        input("press enter to delete created items...")
+        for item in reversed(groups_of_iterest):
+            api.lists.delete(*item)
+
+
+def load_arguments() -> CmdArguments:
+    url = sys.argv[1]
+    port = sys.argv[2]
+    user = sys.argv[3]
+    password = sys.argv[4]
+    return CmdArguments(url, int(port), user, password)
```

Comment:
Can we use `typer` or any other lib for building cmd tools instead of creating from the scratch?

Reply:
I don't want to add non-neccessary dependencies to our library